config/rocketchat_example.yml

```
@@ -0,0 +1,7 @@
+rocketchat:
```

I would see something like `endpoint` here, as both the `endpoint` and the `credentials` belong to Rocket.Chat.
```suggestion
endpoint:
```